File: yukicoder/431_死亡フラグ/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int deadFlag1 = sc.nextInt();
        int deadFlag2 = sc.nextInt();
        int deadFlag3 = sc.nextInt();
        int deadFlagSum = deadFlag1 + deadFlag2 + deadFlag3;

        int survivedFlag = sc.nextInt();

        if (survivedFlag == 1 || deadFlagSum < 2) {
            System.out.println("SURVIVED");
        } else {
            System.out.println("DEAD");
        }

    }

}

File: yukicoder/431_死亡フラグ/Main.cpp
#include <iostream>

using namespace std;

int main() {

    int deadFlag1, deadFlag2, deadFlag3;
    cin >> deadFlag1 >> deadFlag2 >> deadFlag3;

    int deadFlagSum = deadFlag1 + deadFlag2 + deadFlag3;

    int survivedFlag;
    cin >> survivedFlag;

    if (survivedFlag == 1 || deadFlagSum < 2) {
        cout << "SURVIVED" << endl;
    } else {
        cout << "DEAD" << endl;
    }

    return 0;

}
